src/packagedcode/models.py

```diff
@@ -94,14 +103,80 @@
   -- code in source or compiled form or both.
 """
 
-class Package(object):
+
+class EnhancedModel(Model):
+    def __init__(self, deserialize_mapping=None, strict=True, **kwargs):
+        Model.__init__(self, raw_data=kwargs, deserialize_mapping=deserialize_mapping, strict=strict)
+
+
+class Version(EnhancedModel):
+    version = StringType()
+
+
+class AssertedLicense(EnhancedModel):
+    """
+    As asserted in a package
+    """
+    license = StringType(default=None)
```

Comment:
Why do these strings have a default to None and most other strings have no defaults? what is the purpose?